neuro_san/service/grpc/grpc_agent_server.py

```diff
@@ -107,25 +111,14 @@ def get_starting_event(self) -> threading.Event:
         """
         return self.notify_started
 
-    def setup_agent_network_storage(self):
-        """
-        Initialize ServiceAgentNetworkStorage with AgentNetworks
-        we have parsed in server manifest file.
-        """
-        network_storage: ServiceAgentNetworkStorage =\
-            ServiceAgentNetworkStorage.get_instance()
-        network_storage.setup_agent_networks(self.agent_networks)
```

Comment:
Don't really need this method. Just call the network storage directly (see below)